server.js

```diff
@@ -1,27 +1,64 @@
-import express from "express";
+import express, { response } from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Route One - main "page"
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello, let's check out the world of BOOKS!")
+});
+
+// Route Two - All books with query params (title and average_rating)
+app.get("/books", (req, res) => {
+  const { authors, average_rating } = req.query
+  let books = booksData
+  if (average_rating) {
+    books = booksData.filter(rating => rating.average_rating.toLowerCase() === average_rating.toLowerCase());
+  }
+  if (authors) {
+    books = booksData.filter(name => { return name.authors.toLowerCase() === authors.toLowerCase()});
+  }
+
+  res.status(200).json({
+    success: true,
+    message: "OK",
+    body: {
+      booksData: books
+    }
+  });
```

Comment:
When I received my code review this week, it was pointed out that it could be helpful to add another error-msg when - for example - searching for an author that does not exist. So I'll pass this comment on to here too, since the user now only get an empty array + success:true when searching for a slightly misspelled "JK Rowlings", which can be a bit confusing.